pkg/utils/mocks/mock_jira.go

```diff
@@ -0,0 +1,122 @@
+//go:generate mockgen -source=./pkg/utils/jira.go -destination=./mocks/pkg/utils/jira_mock.go -package=utils
```

Comment:
This should be defined where the actual implementation is, not in the mocks file.